lumicks/pylake/tests/test_imaging_camera/test_correlated_stack.py

```diff
@@ -927,3 +927,30 @@ def test_alignment_multistack(rgb_alignment_image_data, gray_alignment_image_dat
             full_stack_idx += 1
 
     assert full_stack.num_frames == len(ref_ts)
+
+
+def test_frame_timestamp_ranges_exclude_false():
```

Comment:
Probably rename the test function?
```suggestion
def test_frame_timestamp_ranges():
```
or
```suggestion
def test_frame_timestamp_ranges_include_true():
```

Reply:
Good point! I'll fixup the name tomorrow.